src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -454,11 +454,11 @@ private function resolveServices($value)
             $value = array_map(array($this, 'resolveServices'), $value);
         } elseif (is_string($value) &&  0 === strpos($value, '@=')) {
             return new Expression(substr($value, 2));
-        } elseif (is_string($value) &&  0 === strpos($value, '@')) {
-            if (0 === strpos($value, '@@')) {
+        } elseif (is_string($value) && $value[0] && '@' === $value[0]) {
```

Comment:
you are still accessing a non-existent index here for the empty string, so triggering a notice. You should use ``'' !== $value``.

and yes, passing the empty string as argument is perfectly valid